membership_file/models.py

```diff
@@ -36,21 +36,28 @@ def get_member(self):
     # Checks whether a given user is a member
     def is_member(self):
         member = self.get_member()
-        return member is not None and not member.is_deregistered
+        return member is not None and member.is_considered_member()
 
 
 ##################################################################################
 
 # The Member model represents a Member in the membership file
 class Member(models.Model):
+
+    class Meta:
+        permissions = [
+            ('can_view_membership_information_self',    "[F] Can view their own membership information."),
+            ('can_change_membership_information_self',  "[F] Can edit their own membership information."),
+        ]
```

Comment:
What is the [F] for?

Reply:
It indicates that this permission only affects the front-end. When assigning permissions, it was unclear which permission affected what specifically (backend, frontend, both?)